src/attributes.jl

```diff
@@ -143,15 +143,17 @@ Return an attribute `attr` of the model `model`.
 
     get(model::ModelLike, attr::AbstractVariableAttribute, v::VariableIndex)
 
-Return an attribute `attr` of the variable `v` in model `model`.
+If the attribute `attr` is set for the variable `v` in the model `model`, return
```

Comment:
Should we mention that `get` should return an error instead of `nothing` for unsupported attributes?